software/auth/auth.go

```diff
@@ -127,6 +131,20 @@ func registryAuth(client houston.ClientInterface, out io.Writer) error {
 	return nil
 }
 
+func getWorkspaces(client houston.ClientInterface, interactive bool) ([]houston.Workspace, error) {
+	var workspaces []houston.Workspace
+	var err error
+
+	if interactive {
+		workspacePageSize := 2
```

Comment:
nit: can we just leave a comment here in the code on why we just need it to be 2 and nothing else

Reply:
sure

Reply:
Done